src/pages/registration/contributors_tab/components/ContributorRow.tsx

```diff
@@ -156,7 +158,14 @@ export const ContributorRow = ({
             flexDirection: 'column',
             gap: '0.25rem',
             alignItems: 'start',
+            paddingY: '0.5rem',
           }}>
+          {!contributor.identity.id && (
+            <Box sx={{ display: 'flex', gap: '0.3rem', alignItems: 'center' }}>
+              <ErrorIcon color="warning" />
+              <Typography fontWeight={700}>{t('registration.contributors.contributor_is_unidentified')}</Typography>
```

Comment:
Noen spesiell grunn til at det er satt `fontWeight={700}` her og ikke `fontWeight="bold`? Ikke at det har noe å si, bare nysjerrig 🤔

Reply:
Tror det bare hang igjen fra at jeg eksperimenterte litt. Lar det være bold for å unngå forvirring!